src/test/java/laddergame/domain/UserTest.java

```diff
@@ -0,0 +1,16 @@
+package laddergame.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+public class UserTest {
+    @DisplayName("유저 생성 테스트")
+    @Test
+    void create_user() {
+        assertThat(new User("pobi").getName().trim()).isEqualTo("pobi");
```

Comment:
```suggestion
        assertThat(new User("pobi").getName()).isEqualTo("pobi");
```

[조건부 (if~else) 로직 테스트코드에서 사용하지 않기](https://jojoldu.tistory.com/642)

테스트 코드는 분명해야합니다. 테스트 코드에 특별한 로직은 피해주는게 좋을것 같아요.